features/unsupported/USBDevice/USBAudio/USBAudio.cpp

```diff
@@ -113,9 +113,12 @@ bool USBAudio::write(uint8_t * buf) {
     return true;
 }
 
-void USBAudio::writeSync(uint8_t *buf)
+void USBAudio::writeSync(uint8_t *buf, int jitter_nb)
 {
-    USBDevice::writeNB(EPISO_IN, buf, PACKET_SIZE_ISO_OUT, PACKET_SIZE_ISO_OUT);
+    if ((jitter_nb != -1) && (jitter_nb !=1)) {
+        jitter_nb = 0;
+    }
+    USBDevice::writeNB(EPISO_IN, buf, PACKET_SIZE_ISO_OUT + jitter_nb *(this->channel_nb_out*2), PACKET_SIZE_ISO_OUT+this->channel_nb_out*2);
```

Comment:
The last argument is changing here from ``PACKET_SIZE_ISO_OUT`` to ``PACKET_SIZE_ISO_OUT+this->channel_nb_out*2`` ?